test/test_rospy/nodes/listener_once.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# Software License Agreement (BSD License)
+#
+# Copyright (c) 2008, Willow Garage, Inc.
```

Comment:
I guess this should be different since the code is not coming from Willow Garage.